doc/man7/provider-digest.pod

```diff
@@ -111,6 +113,9 @@ This function should free any resources associated with that context.
 OSSL_FUNC_digest_dupctx() should duplicate the provider side digest context in the
 I<dctx> parameter and return the duplicate copy.
 
+OSSL_FUNC_digest_copyctx() should copy the provider side digest context in the
+I<vinctx> parameter to the I<voutctx> parameter which is the another provider side context.
+
```

Comment:
Please mention that this function can speed up HMAC operations in the PBKDF2 implementation and that it is optional - dupctx will be used if not present.

Please mention in HISTORY section that this function was added in 3.5 version.